GNUmakefile

```diff
@@ -267,8 +267,12 @@ ui-docker: ui-build-image
 ui-legacy-docker: ui-legacy-build-image
 	@$(SHELL) $(CURDIR)/build-support/scripts/build-docker.sh ui-legacy
 
+test-envoy-integ: dev-docker
+	@$(SHELL) $(CURDIR)/test/integration/connect/envoy/run-tests.sh
```

Comment:
Did you put that into this PR intentionally?

Reply:
Nope 🤦‍♂️